### ISSUE TITLE
Use application-config module to store auth token

### ISSUE THREAD

**Issue by thom4parisot** [dtc-innovation/garagiste#7]
Cf. https://www.npmjs.com/package/application-config

Less code to maintain :-)

**Comment by DavidBruant** [comment]
Yeah, I started with https://developers.google.com/drive/v3/web/quickstart/nodejs?hl=fr without really questionning any of it. It'll come in its time
